app/src/main/java/calculator/view/InputView.java

```diff
@@ -0,0 +1,34 @@
+package calculator.view;
+
+import static calculator.view.OutputView.showMenu;
+
+import calculator.validation.exception.MenuInputException;
+import java.util.Scanner;
+
+public final class InputView {
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final String NUMERIC_REGEX = "\\d+";
+    private static final String NOT_NUMERIC_MENU_INPUT_MESSAGE = "메뉴는 숫자만 입력 가능합니다.";
+
+    private InputView() {
+    }
+
+    public static int inputMenuNumber() {
+        showMenu();
+        String menuInput = readLine().trim();
+        validateIsNumericMenuInput(menuInput);
+
+        return Integer.parseInt(menuInput);
+    }
+
+    public static String readLine() {
+        return SCANNER.nextLine();
+    }
+
+    private static void validateIsNumericMenuInput(String menuInput) {
```

Comment:
네이밍을 조금 제안 드려 보았어요. (반영하시지 않으셔도 됩니다.)
```suggestion
    private static void validateMenuInput(String menu) {
```

Reply:
- InputView 내에서 메뉴입력에 대한 validation은 1개 밖에 없으므로, 제안해주신 네이밍으로 하는 것이 나을 것 같습니다! 감사합니다.